portfolio/src/main/webapp/style.css

```diff
@@ -1,6 +1,7 @@
 #about-me {
-  height: 45vh;
-  margin: 30px auto 50px auto;
+  display: flex;
+  margin: -30px auto 80px auto;
```

Comment:
since we're doing something kind of special here, I'd recommend you add a comment explaining why.

```
/**
 * Negative margin and positive padding allows links in header to scroll to the correct position.
 * See https://stackoverflow.com/a/58890319
 */
```